templates/lxd/index.html

```diff
@@ -0,0 +1,299 @@
+        <li class="p-list__item is-ticked">Projects for segmenting your LXD server, and easy multi-user setup for enhanced security</li>
+        <li class="p-list__item is-ticked">Hardware passthrough (GPU, USB, NIC, disks, and more)</li>
+        <li class="p-list__item is-ticked">Support for live-migration and stateful snapshotting</li>
+        <li class="p-list__item is-ticked">Advanced networking support (OVN, SR-IOV, hardware acceleration  support)</li>
+        <li class="p-list__item is-ticked">High availability clustering when combined with CEPH and OVN for storage and network redundancy</li>
+        <li class="p-list__item is-ticked">LTS releases every two years &mdash; supported for five years</li>
+        <li class="p-list__item is-ticked">Commercial support available through <a href="https://ubuntu.com/pricing/infra">Ubuntu Pro</a></li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <hr class="p-rule is-fixed-width">
+  <div class="row--50-50">
+    <div class="col">
+      <h3 class="p-heading--2">Join the LXD community</h3>
+    </div>
+    <div class="col">
+      <div class="row">
+        <div class="col-2 col-medium-2 col-small-2">
+          <h3 class="p-text--small-caps u-no-margin--bottom">Documentation</h3>
+        </div>
+        <div class="col-4 col-small-2">
+          <p><a href="https://documentation.ubuntu.com/lxd/en/latest/">Read the documentation&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
+      <hr class="p-rule is-muted">
+      <div class="row">
+        <div class="col-2 col-medium-2 col-small-2">
+          <h3 class="p-text--small-caps u-no-margin--bottom">Forum</h3>
+        </div>
+        <div class="col-4 col-small-2">
+          <p><a href="https://discourse.ubuntu.com/c/lxd/126">Discuss with other users&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
+      <hr class="p-rule is-muted">
+      <div class="row">
+        <div class="col-2 col-medium-2 col-small-2">
+          <h3 class="p-text--small-caps u-no-margin--bottom">Tutorials</h3>
+        </div>
+        <div class="col-4 col-small-2">
+          <p><a href="https://www.youtube.com/c/LXDvideos/videos">Watch tutorials on YouTube&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <hr class="p-rule is-fixed-width">
+  <div class="u-fixed-width">
+    {{ image (
+      url="https://assets.ubuntu.com/v1/ca68d7e3-stocksy-woman-working-ar2-1.jpg",
+      alt="",
+      width="2048",
+      height="836",
+      hi_def=True,
+      loading="lazy"
+      ) | safe
+    }}
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="u-fixed-width p-section--shallow">
+		<hr class="p-rule">
+		<h2>Enhance your experience with <br class="u-hide--small" /> best-in-class open-source tools</h2>
+	</div>
+  <div class="row--25-75">
+    <div class="col">
+      <hr class="p-rule">
+      <div class="row">
```

Comment:
It's this way in the design as well, so feel free to ignore, but I think this would look better if we could maintain height consistency as we do elsewhere with this pattern. eg from u.com/ai:
![image](https://github.com/canonical/canonical.com/assets/17607612/04b67777-122c-4fdb-b031-8ff2930e0a7c)